calico_containers/pycalico/datastore.py

```diff
@@ -291,9 +292,31 @@ def ensure_global_config(self):
         # Disable felix status reporting, which consumes etcd write bandwidth.
         self._write_global_config(FELIX_REPORTING_INTERVAL_PATH, "0")
 
+        # Create our Cluster GUID (if it does not already exist).
+        self._ensure_cluster_guid(CLUSTER_GUID_PATH)
```

Comment:
Does `_write_global_config` bail out if the config already exists?  We don't want to update the UUID if it's already there.